.changeset/tiny-lies-type.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/menu': minor
+'@leafygreen-ui/side-nav': minor
```

Comment:
new components should be a major bump so we can get to 1.0.0 when we release; unless we are planning to release this as an alpha/beta like we are doing with MongoNav